config/externalnamenottested.go

```diff
@@ -394,4 +394,101 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 	"aws_elasticsearch_domain_policy": config.IdentifierFromProvider,
 	// Elasticsearch domains can be imported using the domain_name
 	"aws_elasticsearch_domain_saml_options": config.ParameterAsIdentifier("domain_name"),
+
+	// macie2
+	//
+	// aws_macie2_account can be imported using the id
+	"aws_macie2_account": config.IdentifierFromProvider,
+	// aws_macie2_classification_job can be imported using the id
+	"aws_macie2_classification_job": config.IdentifierFromProvider,
+	// aws_macie2_custom_data_identifier can be imported using the id
+	"aws_macie2_custom_data_identifier": config.IdentifierFromProvider,
+	// aws_macie2_findings_filter can be imported using the id
+	"aws_macie2_findings_filter": config.IdentifierFromProvider,
+	// aws_macie2_invitation_accepter can be imported using the admin account ID
+	"aws_macie2_invitation_accepter": FormattedIdentifierFromProvider("", "administrator_account_id"),
+	// aws_macie2_member can be imported using the account ID of the member account
+	"aws_macie2_member": FormattedIdentifierFromProvider("", "account_id"),
+	// aws_macie2_organization_admin_account can be imported using the id
+	"aws_macie2_organization_admin_account": config.IdentifierFromProvider,
+
+	// mediaconvert
+	//
+	// Media Convert Queue can be imported via the queue name
+	"aws_media_convert_queue": config.NameAsIdentifier,
+
+	// mediapackage
+	//
+	// Media Package Channels can be imported via the channel ID
+	"aws_media_package_channel": config.IdentifierFromProvider,
+
+	// mediastore
+	//
+	// MediaStore Container can be imported using the MediaStore Container Name
+	"aws_media_store_container": config.NameAsIdentifier,
+	// MediaStore Container Policy can be imported using the MediaStore Container Name
+	"aws_media_store_container_policy": FormattedIdentifierFromProvider("", "container_name"),
+
+	// memorydb
+	//
+	// Use the name to import an ACL
+	"aws_memorydb_acl": config.NameAsIdentifier,
+	// Use the name to import a cluster
+	"aws_memorydb_cluster": config.NameAsIdentifier,
+	// Use the name to import a parameter group
+	"aws_memorydb_parameter_group": config.NameAsIdentifier,
+	// Use the name to import a snapshot
+	"aws_memorydb_snapshot": config.NameAsIdentifier,
+	// Use the name to import a subnet group
+	"aws_memorydb_subnet_group": config.NameAsIdentifier,
+	// Use the user_name to import a user
+	"aws_memorydb_user": config.ParameterAsIdentifier("user_name"),
+
+	// opsworks
+	//
+	// Opsworks Application can be imported using the id
+	"aws_opsworks_application": config.IdentifierFromProvider,
+	// OpsWorks Custom Layers can be imported using the id
+	"aws_opsworks_custom_layer": config.IdentifierFromProvider,
+	// No import
```

Comment:
Thanks @sergenyalcin for taking note of these TODOs. Let's do our best to enforce these during testing & reviews.